content/en/standard-attributes/_index.md

```diff
@@ -0,0 +1,1807 @@
+---
+title: Default Standard Attributes
+disable_sidebar: true
+further_reading:
+    - link: '/data_security/'
+      tag: 'Documentation'
+      text: 'Ensuring the security of the data sent to Datadog'
+filter_all: All
+content: "The following table lists the attributes automatically applied to data sent to Datadog by the Agent by each of the RUM, Logs, and APM products, as applicable to the data domain. Optionally, filter the list by product or search by keyword or description text to find the attributes you're interested in."
+
+attributes:
```

Comment:
is this data aggregated from somewhere that we can automate?    this is a massive list, just worried it will be difficult to maintain over time.

Reply:
@bgdeutsch It's not, but I would love to. Not sure how to make that happen. Just getting this page made was job 1. (We're already maintaining these manually in each of the product docs pages -- I don't think they change all that often)